### PR TITLE
add a few examples to the JSON editor macro feature

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
It saved me a lot of time when building `Crossed` so I thought it might be a good idea to include the macros I used as examples.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:1126/pr-test (or any other room on that server)

File: client/js/jsonedit.js [hunk 1]
```diff
@@ -307,24 +336,19 @@ const jeCommands = [
...
             const newState = JSON.parse(s);
             macro(newState, variableState);
             batchStart();
-            if(s != JSON.stringify(newState)) {
-              for(const key in w.state)
-                if(newState[key] === undefined)
-                  newState[key] = null;
-              sendPropertyUpdate(id, newState);
-            }
+            await updateWidget(JSON.stringify(newState), s);
```
Comment (ArnoldSmith86) [inline client/js/jsonedit.js:351]: Changing the `id` using macros led to a broken room state. `updateWidget` already takes care of that stuff and this change makes sense anyway.

Comment (RaphaelAlvez) [reply client/js/jsonedit.js:351]: Is this still a draft?

Comment (ArnoldSmith86) [reply client/js/jsonedit.js:351]: Oh. No. The comment doesn't mean it's broken now. It was just meant to explain why I changed it.

**Review by RaphaelAlvez — APPROVED** [review]
Ideally, this should be like in legacy editor and allow us to add more and more premade macros.

code reviewed and tested.
better than the previous situation.

**Comment by ArnoldSmith86** [comment]
> Ideally, this should be like in legacy editor and allow us to add more and more premade macros.

I don't know. I would prefer to add proper commands for stuff we want to use regularly.

**Pull request merged**